da/mock/mock.go

```diff
@@ -33,11 +40,39 @@ type config struct {
 var _ da.DataAvailabilityLayerClient = &DataAvailabilityLayerClient{}
 var _ da.BlockRetriever = &DataAvailabilityLayerClient{}
 
+func getRandomHeader(size int) *core.DataAvailabilityHeader {
```

Comment:
There is also utility on the node that provides this functionality. For now, you may also want to copy it to the rpc.

Reply:
Using `RandEDS, NewDataAvailabilityHeader`.